scenarios/appsec/test_client_ip.py

```diff
@@ -11,7 +11,7 @@
     pytestmark = pytest.mark.skip("not relevant")
 
 
-@released(dotnet="?", golang="?", java="?")
+@released(dotnet="?", golang="?", java="0.114.0")
```

Comment:
I think we are not using 0.114.0 version in this PR checks. We don't know if this change it will work...

Reply:
not in this CI (because we do not have a way to get java artifact without a secret). 

So @smola just need to check locally that the test passes (or the JAVA CI will be brokken).

Reply:
in other words, @smola, if you have checked, it's ok for me !

Reply:
I checked, but anyway, 0.114.0 is now released, so I will retrigger some CI jobs :wink: